src/osd/OpRequest.cc

```diff
@@ -52,9 +52,9 @@ void OpRequest::_dump(Formatter *f) const
   f->dump_string("flag_point", state_string());
   if (m->get_orig_source().is_client()) {
     f->open_object_section("client_info");
-    stringstream client_name;
-    client_name << m->get_orig_source();
-    f->dump_string("client", client_name.str());
+    stringstream client_inst;
+    client_inst << m->get_orig_source_inst();
+    f->dump_string("client", client_inst.str());
```

Comment:
Can you instead add a new field, "addr" or "client_addr", so that "client" is still just teh entity name (client.1234)?